docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -0,0 +1,144 @@
+| Schedule          | ``Schedule<Dynamic>``, ``Schedule<Static>``                               | Specify scheduling policy for work items. ``Dynamic`` scheduling is implemented through a work stealing queue. Default is machine and backend specific. |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| IndexType         | ``IndexType<int>``                                                        | Specify integer type to be used for traversing the iteration space. Defaults to ``int64_t``.                                                            |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| LaunchBounds      | ``LaunchBounds<MaxThreads, MinBlocks>``                                   | Specifies hints to to the compiler about CUDA/HIP launch bounds.                                                                                        |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| WorkTag           | ``SomeClass``                                                             | Specify the work tag type used to call the functor operator. Any arbitrary type defaults to ``void``.                                                   |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Requirements
+~~~~~~~~~~~~
+
+Public Class Members
+--------------------
+
+Constructors
+~~~~~~~~~~~~
+ 
+.. cpp:function:: RangePolicy()
+    
+* Default Constructor uninitialized policy.
+
+.. code-block:: cpp
+
+   template<class ... InitArgs> 
+   RangePolicy(const int64_t& begin, const int64_t& end, const InitArgs ... init_args)
+
+* Provide a start and end index as well as optional arguments to control certain behavior (see below).
+   
+.. code-block:: cpp
+
+   template<class ... InitArgs> 
+   RangePolicy(const ExecutionSpace& space, const int64_t& begin, const int64_t& end, const InitArgs ... init_args)
+   
+* Provide a start and end index and an ``ExecutionSpace`` instance to use as the execution resource, as well as optional arguments to control certain behavior (see below).
```

Comment:
needs indentation

Reply:
why not a cpp function? either maike all cpp::function or none

Reply:
see explanations above